models/skilledImpactRequestsModel.js

```diff
@@ -30,5 +33,40 @@ async function findSkilledImpactRequests(campaign_id) {
         });
 }
 
-module.exports = { findSkilledImpactRequests };
+async function insertSkilledImpactRequests(skilledRequests, campaign_id) {
```

Comment:
Since `insert` can handle multiple rows, I think we can really optimize our database operations here and get this function down from 1 transaction per skilled impact request (and 1 insert per project goal) to 1 transaction with 2 total inserts: one for all skilled impact requests and one for all project goals.

So the function can be structured as a single transaction where we first insert all the skilled impact requests, return the skills and IDs, then reshape the first parameter into an array of project goals that we can pass into `insert` (maybe create an object mapping skills to returned IDs so we can set up references properly, `map` the first parameter into an array of arrays of project goals where each skilled impact request just becomes its array of project goals and each project goal gets the correct ID added as an attribute based on the mapping of the skill from the skilled impact request to a returned ID, flatten the 2D array into a 1D one, then insert that).

CCing @dacruz21.